src/snowflake/connector/ssl_wrap_socket.py

```diff
@@ -39,6 +40,7 @@
 
 def inject_into_urllib3():
     """Monkey-patch urllib3 with PyOpenSSL-backed SSL-support and OCSP."""
+    # TODO replace this with the code in vendored
```

Comment:
nit: still TODO?

Reply:
I'll remove the TODO, I was going to move this into the vendored code, but since I decided not to do this. Having it separate like this makes it easier to update the vendored library. Thanks for catching this, I'll look through the changes to make sure I don't add more TODOs.